modules/tasksUI.js

```diff
@@ -0,0 +1,131 @@
+          taskContainer.id = `${task.id}`;
+          tasksUIelement.appendChild(taskContainer);
+          const taskToogle = document.createElement('input');
+          taskToogle.type = 'checkbox';
+          taskToogle.className = 'task';
+          taskToogle.checked = false;
+          taskToogle.id = `toogle${task.id}`;
+          taskContainer.appendChild(taskToogle);
+          const taskTitle = document.createElement('input');
+          taskTitle.className = 'task-title-text';
+          if (task.completed) {
+            taskToogle.checked = true;
+            taskTitle.style = 'text-decoration-line: line-through;';
+          } else {
+            taskToogle.checked = false;
+            taskTitle.style = '';
+          }
+          taskTitle.id = `title${task.id}`;
+          taskTitle.value = task.description;
+          taskContainer.appendChild(taskTitle);
+
+          taskTitle.addEventListener('change', (event) => {
+            if (taskTitle.value !== task.description) {
+              event.preventDefault();
+              // console.log('pressed Enter');
+              this.editTask(event);
+            }
+          });
+
+          taskToogle.addEventListener('change', (event) => {
+            // console.log(event);
+            // console.log(event.target.id);
+            // const order = event.target.id - 1
+            const dBInstance = JSON.parse(localStorage.getItem('tasksDB'));
+            // console.log(dBInstance);
+            const elementID = document.querySelector(`#${event.target.id}`)
+              .parentElement.id;
+            const taskElement = dBInstance.find(
+              (elem) => elem.id === elementID,
+            );
+            // console.log(taskElement);
+            if (taskElement.completed) {
+              // console.log('checking task is complete');
+              taskToogle.checked = false;
+              taskElement.completed = false;
+              localStorage.setItem('tasksDB', JSON.stringify(dBInstance));
+              TasksUI.loadTasksUI();
+            } else {
+              taskToogle.checked = true;
+              taskElement.completed = true;
+              localStorage.setItem('tasksDB', JSON.stringify(dBInstance));
+              TasksUI.loadTasksUI();
+            }
+          });
+        });
+    }
+  };
+
+  // static removeTask(event) {
+  //   const tasksDbInstance = JSON.parse(localStorage.getItem('tasksDB')) || [];
+  //   const order = parseInt(event.target.id, 10) - 1;
+  //   tasksDbInstance.splice(order, 1);
+  //   localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
+  //   tasksDbInstance.forEach((element) => {
+  //     // console.log(element)
+  //     if (element.index - 1 >= order) {
+  //       element.index -= 1;
+  //       // console.log(element.index);
+  //     }
+  //   });
+  //   localStorage.setItem('tasksDB', JSON.stringify(tasksDbInstance));
+  //   TasksUI.loadTasksUI();
+  // }
```

Comment:
Please remove all unused code as a comment in your code. It makes your code unclean.